presto-tpch/src/main/java/com/facebook/presto/tpch/statistics/TableStatisticsDataRepository.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpch.statistics;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airlift.tpch.TpchColumn;
+import io.airlift.tpch.TpchTable;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static com.facebook.presto.util.Optionals.withBoth;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+
+public class TableStatisticsDataRepository
+{
+    private final ObjectMapper objectMapper;
+
+    public TableStatisticsDataRepository(ObjectMapper objectMapper)
+    {
+        this.objectMapper = objectMapper;
+    }
+
+    public void save(
+            String schemaName,
+            TpchTable<?> table,
+            Optional<TpchColumn<?>> partitionColumn,
+            Optional<String> partitionValue,
+            TableStatisticsData statisticsData)
+    {
+        String filename = tableStatisticsDataFilename(table, partitionColumn, partitionValue);
+        Path path = Paths.get("presto-tpch", "src", "main", "resources", "tpch", "statistics", schemaName, filename + ".json");
```

Comment:
`src` is the ultimate place where these files are supposed to be placed, and leaving transfering of them to humans begs for errors when accessing them (because of misplacement) and is a drag to the humans :). All the devs have source control, so nothing will be unnoticed. Will adding a comment to `RecordTpchTableStatsTool` be an acceptable addressing of this comment?